### ISSUE TITLE
Sorting

### ISSUE THREAD

**Issue by OmlineEditor** [proninyaroslav/blink-comparison#3]
Когда сделаете поля для фото https://github.com/proninyaroslav/blink-comparison/issues/2, прошу добавить имя к фото в основном окне и потом сделать возможность сортировки по имени и по дате фото.

**Comment by proninyaroslav** [comment]
В основном окне имеется ввиду в списке фото?

**Comment by OmlineEditor** [comment]
Да, в овновном окне можно только дату и имя, а когда открываешь фото и изучаешь его там еще видно поле примечание.
Если у вас куча одинаковых пломб без этих полей понять кто где невозможно.
